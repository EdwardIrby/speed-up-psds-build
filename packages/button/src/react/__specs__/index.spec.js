import identity from 'identity-obj-proxy'
import React from 'react'
import renderer from 'react-test-renderer'

import Button from '../index'

test('renders correctly', () => {
  const tree = renderer.create(<Button css={identity}>Label</Button>)
  expect(tree).toMatchSnapshot()
})
