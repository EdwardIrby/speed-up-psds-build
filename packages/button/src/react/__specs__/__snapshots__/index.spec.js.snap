// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button
  className="css-gs8szy"
  disabled={false}
>
  <span>
    Label
  </span>
</button>
`;
